.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Run Ganache with Barge
```

Comment:
2 lines above, we specifically checkout a `v4` branch in barge. I'm in favor for renaming branches on barge so all projects using it in their CI are forced to update their CI flow too